unlock-app/src/__tests__/services/web3Service.test.js

```diff
@@ -1282,6 +1282,44 @@ describe('Web3Service', () => {
 
         web3Service.getKeysForLockOnPage(lockAddress, onPage, byPage)
       })
+
+      it('should get as many owners as there are per page, and not fail on missing keys', done => {
+        expect.assertions(3)
+
+        web3Service._getKeyByLockForOwner = jest.fn(() => {
+          return new Promise(resolve => {
```

Comment:
not important, but this is a good candidate for [`Promise.resolve`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve)